containerize/Containerfile

```diff
@@ -58,10 +62,11 @@ COPY ./containerize/container_default_config.yaml /opt/rapidast/rapidast-default
 COPY ./containerize/path_rapidast.sh /etc/profile.d/rapidast.sh
 
 ### Install RapiDAST requirements, globally, so that it's available to any user
-RUN microdnf install -y --setopt=install_weak_deps=0 java-11-openjdk shadow-utils gtk3 dbus-glib procps git && \
+RUN microdnf install -y --setopt=install_weak_deps=0 java-11-openjdk shadow-utils gtk3 dbus-glib procps git nodejs npm && \
```

Comment:
is `npm` required here ?

Reply:
redocly requires it (see https://github.com/Redocly/redocly-cli/blob/fec8220bef6dc69d8fc57e299f61cc6dac97f9ef/README.md?plain=1#L31).